src/lib/endian.rs

```diff
@@ -0,0 +1,58 @@
+#[cfg(target_endian = "little")]
+pub fn to_le_u64(v: u64) -> u64 {
```

Comment:
These functions exist [in the rust standard library](http://is.gd/WMoDc0), no need to reimplement them.